bin/latest-stable

```diff
@@ -2,4 +2,10 @@
 
 set -euo pipefail
 
-curl -L -s https://dl.k8s.io/release/stable.txt | cut -c 2- -
+query="$1"
```

Comment:
I don't know if they always pass in the filter if there is none - but if they do not pass a parameter:

```suggestion
query="${1:-}"
```

Reply:
Thank you @allen-servedio , may be it is not needed when the script is called via asdf, nonetheless this change makes it more robust. I have committed it together with my fixes.